2024/Autumn/Git-GitHub/code_samples/README.md

```diff
@@ -0,0 +1,20 @@
+# Contributing guidelines
+
+The above sample files contain a bug as mentioned in the issue. Do fix a bug in the files which you can and send a PR to the this repository. Please make sure you write proper commits and commit messages.
```

Comment:
fix  typo 'to the this repo' -> 'to this repo'

Reply:
fixed it